frontend/public/models/yaml-templates.ts

```diff
@@ -162,8 +162,19 @@ spec:
   reportingStart: '2019-01-01T00:00:00Z'
   reportingEnd: '2019-12-30T23:59:59Z'
   runImmediately: true
-`)
-  .setIn([referenceForModel(k8sModels.DeploymentModel), 'default'], `
+`).setIn([referenceForModel(k8sModels.ReportQueryModel), 'default'], `
+apiVersion: metering.openshift.io/v1
```

Comment:
The API version here should match the API version in the model

Reply:
Fixed in both places to 'v1' per @chancez comment